src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
Use instead of `String`  `Color` enum it is critical point coze we can put instead of color `BLACK` -> `BLACH` or `DOG` .
rename it to color (we know that it is ball and it has a color, so there is no needs to repeat it like It is ball ball color)